app/components/preprint-status-banner/template.hbs

```diff
@@ -3,37 +3,44 @@
-                            <a role="button" aria-label="Close" href="#" class="dropdown-toggle" data-toggle="dropdown">
-                                <i class="fa fa-times"></i>
-                            </a>
-                        </div>
-                        <div class="status p-md">
-                            <strong>{{t status}}</strong>
-                            <div>{{t feedbackBaseMessage documentType=submission.provider.documentType}} {{t statusExplanation}}.</div>
-                        </div>
-                        <div class="moderator-comment" aria-labelledby="moderator-feedback">
-                            <p>{{reviewerComment}}</p>
-                            {{#if (not submission.provider.reviewsCommentsAnonymous)}}
-                                <div>{{reviewerName}}</div>
-                            {{/if}}
-                            {{if theme.isProvider theme.provider.name (t "global.brand_name")}} {{t moderator}}
+            {{#if isWithdrawn}}
+                <div class="status-explanation">
+                    <i class="fa {{icon}} status-icon" aria-hidden="true"></i>
+                    <span class="banner-content">{{bannerContent}}</span>
+                </div>
+            {{else}}
+                <div class="status-explanation">
+                    <i class="fa {{icon}} status-icon" aria-hidden="true"></i>
+                    <strong>{{t status}}:</strong>
+                    <span>{{bannerContent}}</span>
+                </div>
+                {{#if (and reviewerComment (not submission.provider.reviewsCommentsPrivate))}}
+                    <div class="dropdown reviewer-feedback">
+                        <button class="btn btn-default dropdown-toggle feedback-btn" type="button" data-toggle="dropdown">
+                            {{t labelModeratorFeedback}} <i class="fa fa-caret-down" aria-hidden="true"></i>
+                        </button>
+                        <div class="dropdown-menu">
+                            <div class="feedback-header">
+                                <span id="moderator-feedback">{{t labelModeratorFeedback}}</span>
+                                <a role="button" aria-label="Close" href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                    <i class="fa fa-times"></i>
+                                </a>
+                            </div>
+                            <div class="status p-md">
+                                <strong>{{t status}}</strong>
+                                <div>{{t feedbackBaseMessage documentType=submission.provider.documentType}} {{t statusExplanation}}.</div>
+                            </div>
+                            <div class="moderator-comment" aria-labelledby="moderator-feedback">
+                                <p>{{reviewerComment}}</p>
+                                {{#if (not submission.provider.reviewsCommentsAnonymous)}}
```

Comment:
Can you add more testing notes. For example, this logic doesn't have any notes as an area where provider config matters

Reply:
The comes from the original template as well. All I did was wrapping them in the `{{else}}` block so that it only shows if the preprint is not withdrawn.